Core/include/Acts/Propagator/Navigator.hpp

```diff
@@ -159,6 +159,10 @@ class Navigator {
 
     /// The tolerance used to defined "reached"
     double tolerance = s_onSurfaceTolerance;
+
+    /// Wether to perform boundary checks for layer resolving (improves
+    /// navigation for bended tracks)
```

Comment:
Can we make this a `BoundaryCheck` here already, rather than a book that's being Auto converted later on?

Reply:
good point

Reply:
```suggestion
    /// navigation for bent tracks)
```